slicedimage/_tocpartition.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from ._imagepartition import ImagePartition
+
+
+class TocPartition(object):
+    def __init__(self, extras=None):
+        self.extras = extras
+        self._tocs = list()
+
+    def validate(self):
+        pass
+
+    def add_toc(self, toc):
+        self._tocs.append(toc)
+
+    def all_tocs(self):
```

Comment:
Just checking: we'll never have more than ~ 1000 TocPartitions? (low end recursion limits)

Reply:
I'll convert this to an iterator.